gui/ThemeEngine.cpp

```diff
@@ -721,6 +724,9 @@ bool ThemeEngine::addBitmap(const Common::String &filename, const Common::String
 		delete surf;
 
 		surf = new Graphics::ManagedSurface(tmp2);
+
+		if (surf->hasTransparentColor())
+			surf->setTransparentColor(surf->getTransparentColor());
```

Comment:
This looks kind of counter intuitive.
Is this really what was intended?

Reply:
Good catch, that was supposed to copy the transparent colour from the unscaled image. It should be fixed now.